test/evergreen.yml

```diff
@@ -565,7 +565,41 @@ functions:
         pip3 install lxml==4.8.0 Pygments==2.11.2 Jinja2==3.0.3 gcovr==5.0
         mkdir -p ../coverage_report
         GCOV=/opt/mongodbtoolchain/v4/bin/gcov gcovr -r .. -f ../src -e '.*/bt_(debug|dump|misc|salvage|vrfy).*' -e '.*/(log|progress|verify_build|strerror|env_msg|err_file|cur_config|os_abort)\..*' -e '.*_stat\..*' -e 'bench' -e 'examples' -e 'test' -e 'ext' -e 'dist' -e 'tools' -j 4 --html-self-contained --html-details ../coverage_report/2_coverage_report.html --json-summary-pretty --json-summary ../coverage_report/1_coverage_report_summary.json
-        python3 ../test/evergreen/code_coverage_analysis.py -c ../coverage_report/1_coverage_report_summary.json -t ../time.txt
+        python3 ../test/evergreen/code_coverage_analysis.py -s ../coverage_report/1_coverage_report_summary.json -t ../time.txt
+
+  "component code coverage analysis":
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/cmake_build"
+      shell: bash
+      script: |
+        set -o errexit
+        set -o verbose
+        virtualenv -p python3 venv
+        source venv/bin/activate
+        pip3 install lxml==4.8.0 Pygments==2.11.2 Jinja2==3.0.3 gcovr==5.0
+        mkdir -p ../coverage_report
+        python3 ../test/evergreen/code_coverage_analysis.py -c component_coverage -s ../coverage_report/1_coverage_report_summary.json -t ../time.txt -o ../atlas_out_code_coverage.json
```

Comment:
Is it necessary to have a new function defined/called, or it's okay to combine this together with the above "code coverage analysis" function?

Reply:
Fixed in 8f643b8384ce228db1a05bad5b326e67a3713d8c. Combined with the existing function